utils/__tests__/dashboard-helper.test.js

```diff
@@ -108,10 +108,6 @@ describe('dashboard-helper', () => {
         'raw-url/dashboards/cool-dash/cool-dash.json',
         { headers: { authorization: 'token token' } }
       );
-      expect(console.log.mock.lastCall[0]).toContain(
```

Comment:
Same comment as the `create-preview-links.test.ts` comment

Reply:
ah, i did mean to do this!  will update and let you know once done

Reply:
done! should be good to go!